src/modules/adaptors/Compound/V3/SupplyAdaptor.sol

```diff
@@ -0,0 +1,177 @@
+// SPDX-License-Identifier: Apache-2.0
+pragma solidity 0.8.21;
+
+import { BaseAdaptor, ERC20, SafeTransferLib, Math, Cellar, Registry } from "src/modules/adaptors/BaseAdaptor.sol";
+import { IComet } from "src/interfaces/external/Compound/IComet.sol";
+
+/**
+ * @title Compound V3 Supply Adaptor
+ * @notice Allows Cellars to supply base token to Compound V3.
+ * @author crispymangoes
+ */
+contract SupplyAdaptor is BaseAdaptor {
+    using SafeTransferLib for ERC20;
+    using Math for uint256;
+
+    //==================== Adaptor Data Specification ====================
+    // adaptorData = abi.encode(Comet comet)
+    // Where:
+    // `comet` is the underling Compound V3 Comet this adaptor is working with
```

Comment:
hmmm it is definitely non standard terminology, but it is clear that `comet` has something to do with Compound V3, and imo I think that is enough for users to go off of and learn more. I dont think its a bad idea to add it, but doing so would mean we should probably add more terms and definitions else where. Like the same user that doesn't know what a comet is, probably doesnt know what an ERC4626 vault is, and how that vault is different from a balancer vault.

Reply:
I think there's a baseline knowledge that we can assume from users if they're digging into the smart contracts: knowing what an ERC4626 since they are dealing with vaults. `Comet` is so one-off / niche to CompoundV3, as it is not used as terminology elsewhere (that I know of) in yield strategies.

I think this change request just lumps up into the natspec update we can tackle with the fix-branch after the audit report comes back to us.